File: public/usage-examples/terminal/write-1-example.py
from splashkit import *

write_line("Progress Bar Simulation:")
write("Loading: [")

for i in range(16):  # Simulate progress in 15 steps
    delay(150)  # Pause for 150 milliseconds
    write("=")  # Append to the progress bar

write_line("] Complete!\n")

File: public/usage-examples/terminal/write/write-1-example.cpp
#include "splashkit.h"

int main()
{
    write_line("Progress Bar Simulation:");
    write("Loading: [");

    for (int i = 0; i <= 15; i++) // Simulate progress in 15 steps
    {
        delay(150); // Pause for 150 milliseconds
        write("="); // Append to the progress bar
    }

    write_line("] Complete!\n");

    return 0;
}